### ISSUE TITLE
Draw order problem with mixes

### ISSUE THREAD

**Issue by DiegoAlvarezP** [orange-games/phaser-spine#8]
Hi, first of all thanks a lot for all the work put into this plugin, works like a charm and it's great to test some spine animations on Phaser.

I came across a strange problem related to animation mix and drawing order.
If I try to mix 2 animations where the draw order of the bones change, sometimes they don't go back to the correct order.

For ex: I have an animation where the character arm moves behind it's body and I try to mix with an animation that pushes the arms to the front of the body, if I change back and forth between this 2 animations, the arms will stay locked either behind or in front of the body.

**Comment by Skeptron** [comment]
Hi DarkRaziel !

Did you find any workaround for your issue? I'm experiencing the same problem.

Thanks!

**Comment by DiegoAlvarezP** [comment]
I actually ended up using spine without mixing animations at all, I never actually found how to fix this problem, and I had no answers at all from any other source.

**Comment by Skeptron** [comment]
Ok. Thank you anyway!

**Comment by NeoHoffa** [comment]
Still having this issue. When cycling through animations the draw order of bones are totally random and not always the same as the first time when it was played. I can't seem to find a method to the madness. Anyone have thoughts on how to fix it?

**Comment by shibekin69** [comment]
You guys tried keying in the draw order at the very first frame of the animations?

**Comment by NeoHoffa** [comment]
Yep, that works sometimes. But not all. Can't pin down the issue when it doesn't work.
